builtin/game/item.lua

```diff
@@ -1,4 +1,22 @@
--- Minetest: builtin/item.lua
+---------------------------------------------------------------------------------
```

Comment:
@BlockMen  lets discuss it in #3390 . And if we change, lets change it for the whole builtin/ directory.